harvard-huit/Harvard-Geospatial-Library
# Removed double header in facets.

**Removed double header in facets.*
* * *


# What does this Pull Request do?
This removes the double "Limit Your Search" header that is above the facets.
 
# How should this be tested?

A description of what steps someone could take to:
* Rebuild your container
* Confirm that you only see 'Limit Your Search' once in the filter bar.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @dl-maura @mferrarini
